### PR TITLE
add querry multiple tables tasks

### PR DESCRIPTION
**Pull request by divinecharlotte**
## IN THIS PR I IMPLEMENTED THE FOLLOWING 
- Created a table named owners with the following columns:
id: integer (set  as autoincremented PRIMARY KEY)
full_name: string
age: integer
empty owners

Create a table named species with the following columns:
id: integer (set  as autoincremented PRIMARY KEY)
name: string
empty species

- **Modified animals table**:
Removed column species
delete species

- Added column species_id which is a foreign key referencing species table
addd specie id

- Added column owner_id which is a foreign key referencing the owner's table
add owner id

- Inserted the following data into the owners table:
Sam Smith 34 years old.
Jennifer Orwell 19 years old.
Bob 45 years old.
Melody Pond 77 years old.
Dean Winchester 14 years old.
Jodie Whittaker 38 years old.
add data in owners

- Inserted the following data into the species table:
Pokemon
Digimon
ADD DATA IN SPECIES

- Modified  inserted animals to include owner information (owner_id):
Sam Smith owns Agumon.
Jennifer Orwell owns Gabumon and Pikachu.
Bob owns Devimon and Plantmon.
Melody Pond owns Charmander, Squirtle, and Blossom.
Dean Winchester owns Angemon and Boarmon.
INCLUDE OWNER ID

- Wrote queries (using JOIN) to answer the following questions:
What animals belong to Melody Pond?
ANIMALS BELONG TO MELODY POND

List of all animals that are pokemon (their type is Pokemon).
POKEM ANIMALS

List all owners and their animals,
OWNERS AND THEIR ANIMALS

List all Digimon owned by Jennifer Orwell.
DIG OWN BY JENIFE

List all animals owned by Dean Winchester that haven't tried to escape.
ANIMALS OWNE DEAN

Who owns the most animals?
WHO OWNS THE MOST ANIMAL

### CODE REVIEW

File: data.sql [hunk 1]
```diff
@@ -29,4 +29,73 @@ VALUES
     ('Angemon', '12/06/2005', 1, TRUE, -45.0),
     ('Boarmon', '07/06/2005', 7, TRUE, 20.4),
     ('Blossom', '13/10/1998', 3, TRUE, 17.0),
-    ('Ditto', '14/05/2022', 4, TRUE, 22.0);
\ No newline at end of file
+    ('Ditto', '14/05/2022', 4, TRUE, 22.0);
```
Comment (Davidosky007) [inline data.sql]: * Kindly use the right Date format `YYYY-MM-DD` for all your date entries as i get an error while trying to run them [Read up on Date formats here ](https://www.w3schools.com/sql/sql_dates.asp)

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
Hi @divinecharlotte 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
* Well written PR description 👍 
* Good looking readme file 👍 
* Descriptive commit message 👍 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @davidsoky007 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: data.sql (hunk 1 above)
Comment (divinecharlotte) [reply data.sql]: thanks @Davidosky007 , added the required changes

**Review by BenMukebo — APPROVED** [review]
### Hi @Davidosky007, 👋 

# STATUS: Approved  🎉 

Great work on making the changes requested by a previous reviewer :1st_place_medal:  
 Your project is complete!  There is nothing else to say other than... it's time to merge it
Congratulations! 🎉  🎉 🎉 




## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @BenMukebo ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**